Compilation Error

I got this error while compiling the tools :

**CMake Error at visualisation/CMakeLists.txt:67 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".**

And the line 67 is 

**qt5_use_modules(sliceViewer Widgets OpenGL Xml)**

I'm using Cmake 3.11. Here are what I obtained when compiling the code
-- -------------------------------------------------------------------------------
-- Checking if DGtal is installed:
--   c++11 enabled by cmake. 
-- DGtal found.
-- Boost version: 1.67.0
-- Found the following Boost libraries:
--   program_options
-- Boost and boost_program_options found.
-- -------------------------------------------------------------------------------
-- Checking if doxygen/dot is installed:
-- dot found: /usr/local/bin/dot. Installing doc in /usr/local/share/DGtalTools
-- configured /Users/NGO_Phuc/Documents/Dgtal/DGtalTools-master/doc/doxy.config.in --> /Users/NGO_Phuc/Documents/Dgtal/DGtalTools-master/newBuild/doxy.config
-- -------------------------------------------------------------------------------
CMake Error at visualisation/CMakeLists.txt:67 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".


-- Configuring incomplete, errors occurred!

--- a/ChangeLog.md
+++ b/ChangeLog.md
@@ -1,3 +1,10 @@
+# DGtalTools 1.0
+
+- *global*:
+    - Continuous integration AppVeyor fix.
+      (Bertrand Kerautret, [##337](https://github.com/DGtal-team/DGtalTools/pull/337)).
+
+
 # DGtalTools 0.9.4
 
 - *converters*

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,7 +1,7 @@
 version: 0.9beta.{build}
 
 environment:
-  BOOST_ROOT: "C:\\Libraries\\boost_1_59_0"
+  BOOST_ROOT: "C:\\Libraries\\boost_1_63_0"
   matrix:
   - VS_GEN: Visual Studio 14 2015
     CONFIG: RelWithDebInfo

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -8,7 +8,7 @@ environment:
     B_NAME: Win32
     BOOST_LIBRARYDIR: "C:\\Libraries\\boost_1_59_0\\lib32-msvc-14.0"
     CONFIGQGL: Release
-    QTDIR: C:\Qt\5.4\msvc2013_opengl 
+    QTDIR: C:\Qt\5.10.1\msvc2013_64
 
 #  - VS_GEN: Visual Studio 14 2015 Win64
 #    CONFIG: Release

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -8,7 +8,7 @@ environment:
     B_NAME: Win32
     BOOST_LIBRARYDIR: "C:\\Libraries\\boost_1_59_0\\lib32-msvc-14.0"
     CONFIGQGL: Release
-    QTDIR: C:\Qt\5.10.1\msvc2013_64
+    QTDIR: "C:\\Qt\\5.6.3\\msvc2013"
 
 #  - VS_GEN: Visual Studio 14 2015 Win64
 #    CONFIG: Release

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -8,7 +8,7 @@ environment:
     B_NAME: Win32
     BOOST_LIBRARYDIR: "C:\\Libraries\\boost_1_59_0\\lib32-msvc-14.0"
     CONFIGQGL: Release
-    QTDIR: "C:\\Qt\\5.6.3\\msvc2013"
+    QTDIR: "C:\\Qt\\5.6.3\\msvc2015"
 
 #  - VS_GEN: Visual Studio 14 2015 Win64
 #    CONFIG: Release

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -6,7 +6,7 @@ environment:
   - VS_GEN: Visual Studio 14 2015
     CONFIG: RelWithDebInfo
     B_NAME: Win32
-    BOOST_LIBRARYDIR: "C:\\Libraries\\boost_1_59_0\\lib32-msvc-14.0"
+    BOOST_LIBRARYDIR: "C:\\Libraries\\boost_1_63_0\\lib32-msvc-14.0"
     CONFIGQGL: Release
     QTDIR: "C:\\Qt\\5.6.3\\msvc2015"
 

--- a/visualisation/CMakeLists.txt
+++ b/visualisation/CMakeLists.txt
@@ -16,8 +16,8 @@ foreach (FILE ${DGTAL_TOOLS_SRC})
 ENDFOREACH(FILE)
 
 if (  WITH_VISU3D_QGLVIEWER )
-
-  SET(DGTAL_EXAMPLES_QGL_SRC
+    set_property(SOURCE ui_sliceViewer.h PROPERTY SKIP_AUTOMOC ON)
+    SET(DGTAL_EXAMPLES_QGL_SRC
     3dVolViewer
     3dVolBoundaryViewer
     3dCurveViewer
@@ -49,7 +49,6 @@ if (  WITH_VISU3D_QGLVIEWER )
   if (WITH_QT4)
     QT4_WRAP_UI(ui_sliceViewer_FORMS_HEADERS sliceViewer.ui)
     QT4_WRAP_CPP(sliceViewer_HEADERS_MOC sliceViewer.h OPTIONS -DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED)
-
     add_executable(sliceViewer sliceViewer.cpp ${sliceViewer_HEADERS_MOC}
                                                ${ui_sliceViewer_FORMS_HEADERS})
     target_link_libraries (sliceViewer ${DGTAL_LIBRARIES}  ${DGtalToolsLibDependencies})
@@ -64,7 +63,6 @@ if (  WITH_VISU3D_QGLVIEWER )
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${Qt5Xml_EXECUTABLE_COMPILE_FLAGS}")
     add_executable(sliceViewer sliceViewer.cpp ${ui_sliceViewer_FORMS_HEADERS})
     target_link_libraries(sliceViewer ${DGTAL_LIBRARIES}  ${DGtalToolsLibDependencies})
-    qt5_use_modules(sliceViewer Widgets OpenGL Xml)
   endif (WITH_QT4)
 
   install(TARGETS sliceViewer RUNTIME DESTINATION bin

--- a/visualisation/sliceViewer.cpp
+++ b/visualisation/sliceViewer.cpp
@@ -473,6 +473,7 @@ int main( int argc, char** argv )
 
   QApplication application(argc,argv);
   Viewer3D<> *viewer = new Viewer3D<>();
+  viewer->show();
   bool usehm = vm.count("hueColorMap");
   bool usegh = vm.count("gradHotColorMap");
   bool usegc = vm.count("gradCoolColorMap");

